frontend/src/data_context_global.tsx

```diff
@@ -149,6 +149,13 @@ export function individualLabel(individual: LimitedIndividual): string {
   return label;
 }
 
+export function breedingLabel(individual: Individual): string {
+  let father = `${individual.father.number}`;
+  let mother = `${individual.mother.number}`;
+  let date = new Date(individual.birth_date)?.toISOString().split("T")[0];
+  return "[" + date + "] " + father + " - " + mother;
+}
+
```

Comment:
See my comment in individual_edit. 
To let this work the same way, we could do 
```suggestion
export function breedingLabel(individual: Individual): string {
  let father = `${individual.father.number}`;
  let mother = `${individual.mother.number}`;
  let date = new Date(individual.birth_date)?.toISOString().split("T")[0];
   return "[" + "Födsel " + date + "] " + father + " - " + mother;
}

```